site/content/events/2015-detroit/_email_proposals.txt

```diff
@@ -0,0 +1 @@
+proposals-cityname-2015@devopsdays.org
```

Comment:
You can set this too. It's `proposals-detroit-2015@devopsdays.org`. It goes to organizers-detroit-2015 by default, although we can change it upon request.